### PR TITLE
Import xrange for python3

### PR DESCRIPTION
**Pull request by paulmaschhoff**
Use six.moves.xrange in peptides.py, since xrange doesn't exist in python3.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -41,7 +41,7 @@
 
   install_requires=[
     'matplotlib',
-    'networkx',
+    'networkx==1.11',
```
Comment (myw) [inline setup.py]: this might be too narrow. persumably `networkx~=1.0` is fine

File: setup.py [hunk 2]
```diff
@@ -41,7 +41,7 @@
```
Comment (myw) [inline setup.py:42]: above, should add `Programming Language :: Python :: 3.5`, and remove the 3.4 unless we test with it.
also should add a `python_requires` key (see the [packaging guide](https://packaging.python.org/guides/distributing-packages-using-setuptools/#python-requires) for reference)

File: .travis.yml [hunk 3]
```diff
@@ -21,6 +21,7 @@ install:
   - make image
 script:
   - make test-tox-ext
+  - docker-compose run --rm pychemy-${TOXENV} pip --version
```
Comment (myw) [inline .travis.yml]: can remove this now

**Review by myw — CHANGES_REQUESTED** [review]
Minor cleanup

File: setup.py (hunk 1 above)
Comment (paulmaschhoff) [reply setup.py]: Already done https://github.com/ginkgobioworks/pychemy/pull/5/commits/90c51dabb6896b6cdefef56a75e6bc034305495b

**Review by myw — APPROVED** [review]
Looks great! Thanks!

**Pull request merged**